breaking-front/src/hooks/queries/useLogout.js

```diff
@@ -0,0 +1,20 @@
+import { getLogout } from 'api/login';
+import { PAGE_PATH } from 'constants/path';
+import { useQuery, useQueryClient } from 'react-query';
+import { useNavigate } from 'react-router-dom';
+
+const useLogout = () => {
+  const queryClient = useQueryClient();
+  const navigate = useNavigate();
+
+  return useQuery(['logout'], getLogout, {
+    enabled: false,
+    onSuccess: () => {
+      alert('로그아웃 되었습니다.');
+      queryClient.resetQueries();
```

Comment:
```suggestion
      queryClient.clear();
```

쿼리 캐시를 지우려면 clear함수를 사용해야 합니다